salt/environment_settings.yml

```diff
@@ -427,27 +365,29 @@ environments:
         - elasticsearch
         - mongodb
         - rabbitmq
+  mitx-staging-qa:
+    business_unit: residential-staging
+    network_prefix: '10.30'
+    vpc_name: MITx QA Staging
+    vpc_peers:
+      - operations-qa
     purposes:
-      mitxonline-production:
-        app: edx-platform
-        business_unit: mitxonline
-        domains:
-          cms: studio.mitxonline.mit.edu
-          lms: courses.mitxonline.mit.edu
-          preview: preview.mitxonline.mit.edu
+      xqwatcher:
+        app: xqueue-watcher
+        business_unit: residential-staging
         versions:
-          <<: *mitxonline_production_versions
-        instances:
-          edxapp:
-            min_number: 6
-            max_number: 10
-            type: r5a.large
-          edx-worker:
-            min_number: 3
-            max_number: 7
-            type: t3a.large
-        security_groups:
-          - edx
+          xqwatcher: 41ef7dff8e73f2ef5a5ffee20c4a5b822eb95341
+          edx_config_repo: https://github.com/edx/configuration
+          edx_config_version: open-release/maple.master
+        courses:
+          - name: 600x
+            num_instances: 2
```

Comment:
Do we need 2 for qa?

Reply:
Probably not. I think we can also remove some of the other watchers for courses that don't use staging.